### PR TITLE
Created UI for listing contacts

### LINKED ISSUE #6: Create list contacts UI

### PR DESCRIPTION
**Pull request by abilash-sajeev**
Fixes #6 

@yedhink Please review

### CODE REVIEW

File: app/javascript/src/components/Dashboard/Contacts/Table.jsx [hunk 1]
```diff
@@ -0,0 +1,84 @@
...
+);
+
+const renderDropdownMenu = () => (
+  <Dropdown buttonStyle="text" icon={() => <MenuHorizontal size={16} />}>
+    <Menu>
+      <MenuItem.Button>Edit</MenuItem.Button>
+      <MenuItem.Button style="danger">Delete</MenuItem.Button>
+    </Menu>
+  </Dropdown>
+);
```
Comment (yedhink) [inline app/javascript/src/components/Dashboard/Contacts/Table.jsx:34]: Let's move these to an `utils.jsx` file within the closest scope. Refer: https://academy.bigbinary.com/learn-react/react-best-practices/structuring-code

File: app/javascript/src/components/Dashboard/Contacts/Table.jsx [hunk 2]
```diff
@@ -0,0 +1,84 @@
...
+            render: formatDate,
+          },
+          {
+            title: "",
+            dataIndex: "action",
+            key: "action",
+            width: "48px",
+            render: renderDropdownMenu,
+          },
+        ]}
```
Comment (yedhink) [inline app/javascript/src/components/Dashboard/Contacts/Table.jsx:77]: Let's move this to either a `constats.jsx` or a utils.jsx as a `buildColumnData` method. The idea is to keep the main logic in the presentation layer very evident while compared to other sub-logic that composes the main JSX logic.

**Review by yedhink — COMMENTED** [review]
@abilash-sajeev

File: app/javascript/src/components/Dashboard/Contacts/Table.jsx (hunk 1 above)
Comment (abilash-sajeev) [reply app/javascript/src/components/Dashboard/Contacts/Table.jsx:34]: Okay. I will make the necessary changes. I have some doubts:
 - Can I link the PR which implements the review comments to the same issue or do i need to create a separate issue for that?
 - Can I fix review comments for multiple PRs in a single PR?

Comment (yedhink) [reply app/javascript/src/components/Dashboard/Contacts/Table.jsx:34]: >Can I fix review comments for multiple PRs in a single PR?

This is fine.

>Can I link the PR which implements the review comments to the same issue or do i need to create a separate issue for that?

Raise a separate issue, add the links to the review comments in that issue and then make a PR fixing all those review comments.

**Pull request merged**